source/blog/2017-07-11-fedora-atomic-26-release.html.md

```diff
@@ -0,0 +1,551 @@
+---
+
+Fedora Atomic 26 is now generally available.  This contains updated package versions to match all of the content in Fedora Server 26, as well as updates to the container platforms.  While we release updates every 2 weeks, this release contains a collection of major improvements including:
+
+* latest rpm-ostree with improvements in package layering
+* default to Overlay2 filesystem for better container storage
+* Docker version 1.13.1
+* Latest versions of Cockpit and Atomic CLI
+
+We're all very excited about the steps forward Atomic Host is taking with this major release.  We hope you're just as excited to try them.  Read on for information about software, upgrading, and more.
+
+READMORE
+
+## Upgrading
+
+This is the first release for Fedora 26 Atomic Host coming only a week after the last release of Fedora 25 Atomic Host. Our next release will be in 3 weeks. Before you [upgrade your Fedora Atomic 25 machines](https://fedoraproject.org/wiki/Atomic_Host_upgrade) there are some things you should know.
+
+* Please make sure your root filesystem has a few gigabytes of free space. If your root filesystem never got extended to larger than 3GiB (the default on AWS), then you should resize it before attempting an upgrade.
+
+* If you use the Kubernetes software within Atomic Host then please wait until our next release of Atomic Host to upgrade. There is a [bug that is preventing the API server from starting](https://bugzilla.redhat.com/show_bug.cgi?id=1441218). If you don't rely on the built-in Kubernetes (e.g. you run containerized Kubernetes or OpenShift) this should not affect you. Before the next release we will publish a guide to upgrading your atomic host and making any configuration changes for the new version of kubernetes.
+
+* The storage driver now defaults to overlay2 instead of devicemapper for container storage.  This will not change existing installations.
+
+* The url for the F26 ostree 'remote' is [https://kojipkgs.fedoraproject.org/atomic/26/](https://kojipkgs.fedoraproject.org/atomic/26/)
+
+* We updated our ref for this release to shorten it some and remove the mention of a specific container runtime. In Fedora 25 we had `fedora-atomic/25/x86_64/docker-host`. In Fedora 26 the ref we now have is`fedora/26/x86_64/atomic-host`.
+
+* Because of the way the Fedora release processes work with
+  regards to freezing around major release time some versions
+  of software from the last release of Fedora 25 to this first
+  release of Fedora 26 go backwards. The next two week release
+  will not have that problem.
+
+We will write blog posts and documentation over the next few weeks
+to show you new features and guide how to upgrade your existing hosts.
+
+## Updated Software
+
+This release contains the following changes since last release:
+
+* ostree diff commit old: ce555fa89da934e6eef23764fb40e8333234b8b60b6f688222247c958e5ebd5b
+* ostree diff commit new: 2e4fcaff52c18cf705a2ecfa47d2ca60958f7443177974f24cc9ce07494b3317
```

Comment:
We usually put all the text in the release blog, partly because in the email it's split over two emails.